native/plugin/plugin_integration_test.go

```diff
@@ -31,15 +30,12 @@ func TestCallingPluginFunctionWithReturnValueNoRaceDetection(t *testing.T) {
 	input := `
 	require "plugin"
 
-	p = Plugin.use "../test_fixtures/import_test/plugin/plugin.go"
+	p = Plugin.use "../test_fixtures/import_test.VM()/plugin.go"
```

Comment:
Why changing the path, did you move the file as well?

Reply:
bad search and replace